can_tools/scrapers/base.py

```diff
@@ -76,6 +114,15 @@ class DatasetBase(ABC):
     base_path: Path
     source: str
 
+    # this list of tuples specifies categories for which the first
+    # category listed should always be less than or equal to the second
+    # category listed, when the measurement type is cumulative
+    ordered_validation_categories = [
+        ("total_vaccine_initiated", "total_vaccine_doses_administered"),
+        ("total_vaccine_completed", "total_vaccine_doses_administered"),
+        ("total_vaccine_completed", "total_vaccine_initiated"),
```

Comment:
Did we consider:

```
("total_vaccine_doses_administered", "total_vaccine_distributed")
```

?  Though I recall there being a bit of weirdness where we were sometimes getting 5 doses out of 4-dose vials or whatever...  so maybe it'll be noisy.  Happy to leave it out for now.

Reply:
Good thought. I didn't think about that one because we don't have a whole lot of `total_vaccine_distributed` data

Probably (I hope!) wouldn't hurt to include because any scrapers that don't have both variables would just move past that check